### ISSUE TITLE
Grafik 2.0.0 Ballot

### ISSUE THREAD

**Issue by lpg-tech** [hl7ch/ch-lab-order#322]
eMail Corina von Känel 5/27/24, 8:51 AM:

Ich würde mir wünschen, dass das neue CH LAB Order im Grundsatz in einer Übersicht aufzeigen würde, welche Elemente verwendet werden mit Beispielinhalten. So kann jeder der damit arbeitet rasch eine Übersicht gewinnen, ohne viele Seiten von Text zu lesen. Ansprechende Beispiele gibt es im Internationalen FHIR «ORDER Catalog Implementation Guide»

 

https://build.fhir.org/ig/HL7/fhir-order-catalog/exlabservices.html#LabE6

**Comment by lpg-tech** [comment]
Die erwähnte Grafik bezieht sich auf den Order Catalog, der in lab-order nicht verwendet wird.

**Comment by CorinavonKaenel** [comment]
Genau, diese gibt es nicht. Es ist aber wünschenswert ein solches zu haben und dass alle Beispiele diesen Aufbau dann aufweisen

**Comment by lpg-tech** [comment]
Hier ist die Übersichtsgrafik:
Use Case 5: Building laboratory order panels. These panels instantiate Plandefinition from Order Catalog
